ideascube/mediacenter/management/commands/remove_medias.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import argparse
+from os.path import join as pj
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from ideascube.mediacenter.models import Document
+from ideascube.utils import printerr
+
+
+class Command(BaseCommand):
+    help = ('Remove files from the mediacenter.')
+
+    def add_arguments(self, parser):
+        subs = parser.add_subparsers(
+            title='Commands', dest='cmd', metavar='',
+            parser_class=argparse.ArgumentParser)
+        remove = subs.add_parser(
+            'clean_orphans',
+            help='Clean mediacenter files not associated with a document.')
+        remove.set_defaults(func=self.clean_orphans)
```

Comment:
Hum, this is something I wanted to explain in the PR message but I've forget.

The ideas is to add another subparser to the remove_medias command to solve #685.
I really should have explain it in the PR message and you properly point it in several other comments (I will not answer them).

I concede it complicate a bit this PR and I can remove it if you think it's better.
Having two sub-commands may be just not the right solution. This have to be discuss but if we agree we must have two separated command, there should be no subparser here.